build/DeployTasks/DeployVsixToMarketplaceTask.cs

```diff
@@ -23,7 +23,7 @@ public override bool ShouldRun(BuildContext context)
         return false;
     }
 
-    public override void Run(BuildContext context)
+    public override async void Run(BuildContext context)
```

Comment:
@harry-cpp should we really be using `async void` ? Is there not an `async Task Run(...` we can override?
`async void` has "issues"

Reply:
I don't believe there is one :(